.gitignore

```diff
@@ -163,3 +163,6 @@ _work/
 
 # OS stuff
 .DS_store
+
+# written by setuptools_scm
+**/_version.py
```

Comment:
This is confusing as the projects don't seem to be using `setuptools_scm` (yet?).

Reply:
yet! indeed, but will after the first release :)